src/models/movies-model.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+const fs = require('fs');
+const path = require('path');
+const uuidv4 = require('uuid/v4');
+const { promisify } = require('util');
+
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
+const JSON_PATH = path.join(__dirname, '../../data.json');
+
+class MoviesModel {
+  constructor(jsonPath = JSON_PATH) {
+      this.jsonPath = jsonPath;
+  }
+
+  async getList() {
+      const fileContent = await readFile(this.jsonPath);
+
+      return (
+        JSON.
+        parse(fileContent).
```

Comment:
indentation, also parse can be on the same line

Reply:
> indentation, also parse can be on the same line

refactored